src/tools/cephfs/cephfs-shell

```diff
@@ -939,6 +939,8 @@ class CephFSShell(Cmd):
         """
         Remove a specific file
         """
+        errors = {2: 'No such file or directory',
+                  21: 'Is a directory'}
```

Comment:
I do understand the concern here for adding a custom dictionary but if we add it for every command it will eventually become a  mess. Better to avoid it for now and make this change in single go for entire cephfs-shell codebase later on a separate PR.

Reply:
Removed